docs/SUMMARY.md

```diff
@@ -27,6 +27,8 @@
     * [Datatypes](engine/datatypes.md)
     * Formats
         * [ZenGin Animations](engine/formats/animation.md)
+        * [ZenGin DAT](engine/formats/dat.md)
+            * [Daedalus Bytecode](engine/formats/bytecode.md)
```

Comment:
I think this should be an empty section header named "ZenGin Scripts" and its subpages should be "Daedalus Bytecode" and "Script Binaries" to leave room for possible future documentation on the Daedalus language.

Reply:
Done